auto3dseg/algorithm_templates/dints/scripts/dummy_runner.py

```diff
@@ -68,12 +68,22 @@ def __init__(self, output_path, data_stats_file, device_id: int = 0):
         pixdim = parser.get_parsed_content("transforms_train#transforms#4#pixdim")
         pixdim = [np.abs(pixdim[_i]) for _i in range(3)]
 
-        if "sizemm" not in data_stat["stats_summary"]["image_stats"]:
-            raise ValueError("The data stats file is generated by older version of MONAI. "
-                             "Please update MONAI >= 1.2 and re-run the data analyzer on your dataset.")
-
-        image_size_mm = data_stat["stats_summary"]["image_stats"]["sizemm"]["percentile_99_5"]
-        self.max_shape = [int(np.ceil(image_size_mm[_l] / pixdim[_l])) for _l in range(3)]
+        self.max_shape = [0, 0, 0]
```

Comment:
These lines reverted my previous PR: https://github.com/Project-MONAI/research-contributions/pull/208. Please double check if the merge conflicts are resolved properly.